From 34ea7e9e0420b96aff4a44b5f230413ca941939b Mon Sep 17 00:00:00 2001
From: Artom Lifshitz <alifshit@redhat.com>
Date: Tue, 24 May 2022 10:48:56 -0400
Subject: [PATCH 1/1] Make test_server_actions.resource_setup() wait for
 SSHABLE

Continuing the work of 4fc47f68c7710ccfb67c3ad28a58e047a8b51c1a, we
missed a spot in the recreate_server() call from resource_setup().
Start passing wait_until='SSHABLE' in that call.

Change-Id: I5c6d9afa1148772537d69a2a3321fcf32ad7eea5
---
 tempest/api/compute/servers/test_server_actions.py | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/tempest/api/compute/servers/test_server_actions.py b/tempest/api/compute/servers/test_server_actions.py
index bd4e0e829..0ed73a8f4 100644
--- a/tempest/api/compute/servers/test_server_actions.py
+++ b/tempest/api/compute/servers/test_server_actions.py
@@ -92,7 +92,8 @@ class ServerActionsTestJSON(base.BaseV2ComputeTest):
     @classmethod
     def resource_setup(cls):
         super(ServerActionsTestJSON, cls).resource_setup()
-        cls.server_id = cls.recreate_server(None, validatable=True)
+        cls.server_id = cls.recreate_server(None, validatable=True,
+                                            wait_until='SSHABLE')
 
     @decorators.idempotent_id('6158df09-4b82-4ab3-af6d-29cf36af858d')
     @testtools.skipUnless(CONF.compute_feature_enabled.change_password,
-- 
2.36.1

